src/devices/Is31fl3731/BreakoutRgb5x5.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.Threading;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Represents 5x5 RGB matrix, driven by an IS31FL3731 LED chip.
+    /// </summary>
+    // Product: https://shop.pimoroni.com/products/5x5-rgb-matrix-breakout
+    public class BreakoutRgb5x5 : Is31fl3731
+    {
+        private byte[][] _addresses = new byte[][]
```

Comment:
maybe even static readonly? So less chances to "damage" it and as it's anyway like a constant, a bit less memory footprint when it's used?